src/main/resources/application.conf

```diff
@@ -38,10 +39,10 @@ variant-gene {
 
 variant-disease {
   path = ${output}"/v2d/"
-  studies = ${input}"/v2d/200207/studies.parquet"
-  toploci = ${input}"/v2d/200207/toploci.parquet"
-  finemapping = ${input}"/v2d/200207/finemapping.parquet"
-  ld = ${input}"/v2d/200207/ld.parquet"
-  overlapping = ${input}"/v2d/200207/locus_overlap.parquet"
-  coloc = ${input}"/coloc/190612/"
+  studies = ${input}"/v2d/210601/studies.parquet"
```

Comment:
Should all those '210601' references be a variable or could they change independently?

Reply:
@JarrodBaker ideally it should all come from the same run but it is realistically a bit more difficult. you don't need to run the whole input data and can generate partial runs as in the genetic case, there are no new QTL datasets nor functional genomic ones just new GWASes.